answerking_app/tests/fixtures/products/basic-1.json

```diff
@@ -3,6 +3,6 @@
     "name": "Burger",
     "description": "desc",
     "price": 1.2,
-    "category": [],
```

Comment:
These are used by the integration tests and will be replaced so dont' worry about changing these, just leave them as they are and I will change them in my PR.